sound/soc/sof/intel/hda-dsp.c

```diff
@@ -433,6 +433,19 @@ int hda_dsp_runtime_resume(struct snd_sof_dev *sdev)
 	return hda_resume(sdev);
 }
 
+int hda_dsp_runtime_idle(struct snd_sof_dev *sdev)
+{
+	struct hdac_bus *hbus = sof_to_bus(sdev);
+
+	if (hbus->codec_powered) {
```

Comment:
question: have you tested this in nocodec mode or when SOF_HDA is not defined? Or verified that this field is properly zeroed and this is harmless

Reply:
@plbossart Ack, this was tested. The field is part of hdac_bus which is allocated and set to zero for all cases in hda.c::hda_dsp_probe probe via
```
hdev = devm_kzalloc(sdev->dev, sizeof(*hdev), GFP_KERNEL);
´´

... this is done for nocodec as well, so we are good.